common/scripted_effects/00_scripted_effects.txt

```diff
@@ -5230,6 +5230,11 @@ remove_symptoms_effect = {
 		limit = { trait = malaise}
```

Comment:
Dark Healing stopped working. Also, the flags not cleared.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79054301-d15f4400-7c54-11ea-8edb-0d3368b77494.png)
![image](https://user-images.githubusercontent.com/46384992/79054310-da501580-7c54-11ea-8e78-52266ae63139.png)

</details>